Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/Utils/HttpHandlerHelper.cs

```diff
@@ -39,7 +41,19 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 }
             }
 
-            httpResponse =  await base.SendAsync(request, cancellationToken);
+            try
+            {
+                httpResponse = await base.SendAsync(request, cancellationToken);
+            }
+            catch (Exception ex) {
+
+                if (this.ExceptionIntercepter == null)
+                {
+                    throw;
+                }
+                this.ExceptionIntercepter?.Invoke(request, ex);
```

Comment:
There is already a check for null above, at this point, we know the variable is not null, right?

```suggestion
                this.ExceptionIntercepter.Invoke(request, ex);
```